nservicebus/licensing/index.md

```diff
@@ -15,7 +15,7 @@ See [Particular Licensing](http://particular.net/licensing) for specific license
 
 Supported versions include any major version released within the last 3 years. Within a supported major version, we support the latest minor version, or any minor version released within the last year.
 
-If your version is not listed or has expired but you would like support, please contact us. For more information on the relationship between licensing and release see our [release policy](/nservicebus/upgrades/release-policy.md).
+If your version is not listed or has expired but you would like support, please [contact us](http://particular.net/contactus). For more information on the relationship between licensing and release see our [release policy](/nservicebus/upgrades/release-policy.md).
```

Comment:
How about:
If your current version has expired and you require support, please [contact us](http://particular.net/contactus). For more information please read our [release policy](/nservicebus/upgrades/release-policy.md).